playground/src/SyntaxStyler/syntaxStyles.css

```diff
@@ -0,0 +1,19 @@
+.tsdoc-underline {
+  text-decoration: underline !important;
+}
+
+.tsdoc-blocktag {
+  border-width: 1px;
+  border-top-style: solid;
+  border-left-style: solid;
+  border-right-style: none;
+  border-bottom-style: none;
+  border-top-left-radius: 5px;
```

Comment:
>5px [](start = 26, length = 3)

Reduce to 3px...  it should be subtle :-P

Reply:
Also I think it looks better with `border-color: #a6a6a6;`

---
In reply to: [223970835](https://github.com/Microsoft/tsdoc/pull/93#discussion_r223970835) [](ancestors = 223970835)